numdifftools/nd_algopy.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, fun, n=1, method='forward', full_output=False):
 
     @property
     def fun(self):
+        o = x.test()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unused local variable "o". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://35.168.58.35:9000/coding_rules#rule_key=python%3AS1481)